castle/cms/profiles/2_2_0/registry/controlpanel.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0"?>
+<registry xmlns:i18n="http://xml.zope.org/namespaces/i18n"
+          i18n:domain="plone">
+  <records interface="castle.cms.interfaces.ISecuritySchema"
```

Comment:
To upgrade only a specific field, you can do something like what is done with `file_upload_fields` instead. I imagine this doesn't overwrite any settings though so it's probably fine. However, it might be nice being explicit here so we know what the upgrade is targeting.

Reply:
I think I'm going to leave it as-is, since adding all the extra details when it's provided in `ISecuritySchema` already is redundant. I will, however, add a comment that specifies what change occurred. Is that a fair compromise?

Reply:
Sounds good!